toolkit/analysis_and_visualization/classification/assets/scripts/app.js

```diff
@@ -0,0 +1,578 @@
+/*global data, labelSummary, imageSummary, jQuery, $, modelHistories*/
```

Comment:
Codacy found an issue: ['jQuery' is already defined as a built-in global variable.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)